### DISCUSSION TITLE
needs-of-self-archiving

### DISCUSSION THREAD

**Discussion by giscus** [sangwoo-joh/reactions#14] [Comments]
# needs-of-self-archiving

문득 세어보니 내 크롬 북마크가 약 600여개 정도 된다. 이 중 대부분은 나중에 읽으려고 일단 추가해둔 것들이다. 이것도 일종의 technical debt라고 할 수 있을까?

https://sangwoo-joh.github.io/needs-of-self-archiving

**Comment by sangwoo-joh** [comment]
번외로 giscus가 글을 쓰자마자 discussion이 자동으로 생성되는게 아니라, 최소 한번은 리액션 또는 댓글을 남겼다가 지워야 생성되는 구조인 것 같은데... 트리거 타이밍을 조절할 수는 없는지 살펴봐야겠다.

**Reply by kkeundotnet** [reply]
제가 그런 이유로 좋아요를 못 누르고 있었는데 어찌 아시고 ㅋㅋㅋ

**Comment by kkeundotnet** [comment]
> 문득 세어보니 내 크롬 북마크가 약 600여개 정도 된다. 이 중 대부분은 나중에 읽으려고 일단 추가해둔 것들이다. 이것도 일종의 technical debt라고 할 수 있을까?

이건 그냥 가벼운 저장강박 같은 것 아닌가요? 600개나 쌓이셨으면 이미 읽기 속도보다 모으기 속도가 한참 빠른 것 같은데, 속도 조절 안 하시고 그대로면 앞으로도 단조증가합니다. （⌒▽⌒ゞ

**Comment by sangwoo-joh** [comment]
김창준님의 애자일 관련 글을 보려고 들어갔더니 이글루스도 2023년에 문을 닫았고, 혹시나 웹에 아카이빙된 것은 없나 하고 살펴봤더니 구글의 저장된 페이지 기능도 2024년에 종료되었다. 안타깝다. 당분간은 내 아카이빙 볼륨을 키우는데 주력해야 겠다.
